dotnet/src/webdriver/Chromium/ChromiumDriver.cs

```diff
@@ -43,6 +43,7 @@ public abstract class ChromiumDriver : WebDriver, ISupportsLogs, IDevTools
         private const string DeleteNetworkConditionsCommand = "deleteNetworkConditions";
         private const string SendChromeCommand = "sendChromeCommand";
         private const string SendChromeCommandWithResult = "sendChromeCommandWithResult";
+        protected const string ExecuteCdpCommand = "executeCdpCommand";
```

Comment:
Change this to something arbitrary. Keep the value the same (`"executeCdpCommand"`), but just change the field name. Maybe to `ExecuteCdp`, or `ExecuteDevToolsCommand`.